handsontable/src/3rdparty/walkontable/src/selection/selection.js

```diff
@@ -0,0 +1,111 @@
+import { mixin } from '../../../../helpers/object';
+import localHooks from '../../../../mixins/localHooks';
+
+/**
+ * The Selection class allows highlighting (by applying CSS class) the table's cells or headers
```

Comment:
Is this description correct? If I'm reading this class correctly, it doesn't apply any CSS classes; this seems to be done in the selection manager.

Reply:
Generally, it's true. The Selection class as a protected API is used by Handsontable to control where (based on the coords) the CSS classes (passed in constructor `new Selection(cellRange, { className: 'my-css-class' })`) should be applied. From the side of Handsontable as an API consumer, it seems to make sense 🤔